src/styles/components/_speaker.scss

```diff
@@ -0,0 +1,38 @@
+.c-speaker {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 300px;
```

Comment:
The speaker card dimensions don't seem accurate to what the design says they are
<img width="756" alt="Screenshot 2023-08-02 at 21 29 03" src="https://github.com/geektutor/devfestlagos2023-web/assets/36075575/0b48e5dc-b99e-4be3-9312-8fa77f573b9f">

But that aside I'd advise not setting a direct height and width for the cards and instead letting the content define the size of the card. Eg you set the image width and use the ratio() mixin in the `mixins.scss` file to make the height and width conform. You can see examples of me using it in the `_home.scss` file. You'll notice that I didn't have to set the image heights when making the page responsive because once I set the width the height would match up to that.